ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo/Byron.hs

```diff
@@ -92,25 +95,34 @@ delegateNodeIdPbft dlg vk = join $ certificateNodeIdPbft dlg <$> findCertByDeleg
 checkDelegationSubject :: Cardano.VerificationKey -> Cardano.Delegation.Certificate -> Bool
 checkDelegationSubject vk = (== vk) . Cardano.Delegation.delegateVK
 
+-- | See chapter 4.1 of
+--   https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/byronChainSpec/latest/download-by-type/doc-pdf/blockchain-spec
+mainnetPBftSignatureThreshold :: PBftSignatureThreshold
+mainnetPBftSignatureThreshold = PBftSignatureThreshold 7
```

Comment:
Wait, what? Where did this come from? It's 0.22 isn't it?

Reply:
I fixed that in a later commit!